recipe/patches/0002-Use-a-standard-autoconf-setup.patch

```diff
@@ -41,9 +42,9 @@ index 1aab3453..f39a0af9 100644
  
 +AC_CONFIG_MACRO_DIRS([cnf/m4])
```

Comment:
I would suggest to tweak this patch and insert `AC_CONFIG_MACRO_DIRS` after `AC_CONFIG_AUX_DIR` -- that way the diff here would not involve the GAP package version and thus would not need to be updated for each GAP release

Reply:
Removed this patch